scripts/deploy-rufu-073.mjs

```
@@ -0,0 +1,293 @@
+ */
+export function resolveRestartAction(cmdline, repoRoot) {
+  if (typeof cmdline !== "string" || cmdline.length === 0) return "skip";
+  return isProvenFusionDaemon(cmdline, repoRoot) ? "signal" : "refuse";
+}
+
+/** Best-effort read of a pid's command line via /proc (POSIX) then `ps -o args=`. Null when unreadable. */
+export function readCmdline(pid) {
+  if (!Number.isInteger(pid) || pid <= 0) return null;
+  try {
+    const data = readFileSync(`/proc/${pid}/cmdline`, "utf8");
+    if (data) return data.replace(/\u0000/g, " ").trim();
```

ESLint

[error] Unexpected control character(s) in regular expression: \x00.

(no-control-regex)